site/src/pages/HealthPage/DERPRegionPage.tsx

```diff
@@ -199,4 +156,68 @@ export const DERPRegionPage: FC = () => {
   );
 };
 
+type StatusCircleProps = { color: string };
+
+const StatusCircle: FC<StatusCircleProps> = ({ color }) => {
+  return (
+    <div
+      css={{
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+      }}
+    >
+      <div
+        css={{
+          width: 8,
+          height: 8,
+          backgroundColor: color,
+          borderRadius: 9999,
+        }}
+      />
+    </div>
+  );
+};
+
+const reportStyles = {
+  header: {
+    padding: 24,
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  title: {
+    fontWeight: 500,
+    margin: 0,
+    lineHeight: "1",
+  },
+  pills: {
+    display: "flex",
+    gap: 8,
+    alignItems: "center",
+  },
+  ports: (theme) => ({
+    display: "flex",
+    alignItems: "center",
+    gap: 8,
+    color: theme.palette.text.secondary,
+    fontSize: 12,
+    lineHeight: "120%",
```

Comment:
I know this was in the code before, but could you swap this out for `1.2`, just to be on the safe side, and avoid weird text leading issues?